gips/data/core.py

```diff
@@ -701,8 +701,6 @@ def open_assets(self, product):
     # TODO - make general product_filter function
     def masks(self, patterns=None):
         """ List all products that are masks """
-        if patterns is None:
-            patterns = ['acca', 'fmask', 'mask']
         m = []
         for p in self.products:
             if any(pattern in p for pattern in patterns):
```

Comment:
Ooooh....I see, so it may need to be specified here, but can be defaulted to `None` in `landsatData.masks`.  I believe the existing behaviour is to have it get driver defined masks and apply them when `patterns is None`.

I believe that `masks` is probably not covered by any tests, so I'll add unit tests to this as well.

Thanks.